xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1549,6 +1549,9 @@ core.recyclebin.confirm=Are you sure you wish to move this page to the recycle b
 core.recyclebin.confirmWithInlinks=Are you sure you wish to move this page to the recycle bin?
 core.recyclebin.completelyDeleteConfirm=This action is not reversible. Are you sure you wish to continue?
 core.recyclebin.invalidEntry=Invalid recycle bin entry.
+core.recyclebin.wantToSkip=Are you sure you wish to delete this page?
+core.recyclebin.wantToSkip.no=Delete and move to the recycle bin.
+core.recyclebin.wantToSkip.yes=Permanently delete the page (it won't be put in the recycle bin).
```

Comment:
Based on the translations alone from what I understand you can only:
  - delete to the recycle bin
  - delete completely

You cannot just cancel the deletion? Could be good to have a screenshot.

Reply:
The delete and cancel buttons are already defined prior to this evolution. For a screenshot, see https://forum.xwiki.org/uploads/default/original/2X/4/4334e3d53f1068d9156031b7b1f9c63d30e608d3.png, from the forum discussion https://forum.xwiki.org/t/xwiki-17401-option-for-deleting-documents-and-or-attachments-without-sending-to-recycle-bin/

Reply:
Ok thanks for the links.